src/main/java/com/jalasoft/selenium/Ariel/movies/Customer.java

```diff
@@ -0,0 +1,91 @@
+package com.jalasoft.selenium.Ariel.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * Created by Ariel Mattos on 3/6/2017.
+ */
+
+class Customer {
+    private String name;
+    private Vector rentals = new Vector();
+
+    /**
+     * Creates a Children movie with the given title.
+     * @param name          the name of the customer.
+     */
+    Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Add a movie rental object to the list of the user's movie rentals.
+     * @param rental        a movie rental object.
+     */
+    public void addRental(final Rental rental) {
+        rentals.addElement(rental);
+    }
+
+    /**
+     * Returns the name of the user.
+     * @return              a String containing the name of the user.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Returns a statement of the movie rentals, total charges and frequent renter points.
+     * @return               a string containing the movie rentals, total charges and frequent renter points.
+     */
+    public String statement() {
+        Enumeration rentals = this.rentals.elements();
+        StringBuffer report = new StringBuffer();
+        report.append("Rental Record for " + getName() + "\n");
+        while (rentals.hasMoreElements()) {
+            Rental each = (Rental) rentals.nextElement();
+            //show figures for this rental
+            String rentalFigure = "\t" + each.getMovie().getTitle()
```

Comment:
It is not a good practice to concatenate string with +
take advantage of StringBuilder class